vpnglean/glean_parser_ext/templates/cpp.jinja2

```diff
@@ -30,15 +30,13 @@ namespace {{ category_name|snake_case }} {
 {% endfor %}
 } // namespace mozilla::glean
 
-namespace {
 {% for category_name, objs in all_objs.items() %}
   {% for obj in objs.values() %}
   {% if obj|attr("_generate_enums") %}
 {{ macros.generate_extra_keys_parser(obj, category_name) }}
   {%- endif %}
   {% endfor %}
 {% endfor %}
-} // namespace
```

Comment:
Had to remove the namespace because this is now a property on the GleanMetrics class. Added the `__DONOTUSE__` prefix, just in case people see it in the autocomplete and get confused.